### PR TITLE
Login button

### PR DESCRIPTION
**Pull request by davidpmccormick**
## Who is this for?
People who want to sign into their library account

## What is it doing for them?
Moving the login to a consistent location in the header

I duped the `Header` component to create `HeaderPrototype` with the required amends to make this work, so that the amount of toggle logic could be kept to a minimum. Since the Header isn't likely to undergo much if any change while we figure this out, this felt like the right way to go.

Behind the `showLogin` toggle.

__Big__
Screenshot 2021-07-14 at 14 48 29

__Medium__
Screenshot 2021-07-14 at 14 48 46

__Small__
Screenshot 2021-07-14 at 14 49 13

### CODE REVIEW

File: common/views/components/PageLayoutDeprecated/PageLayoutDeprecated.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ import Head from 'next/head';
 import convertUrlToString from '../../../utils/convert-url-to-string';
 // $FlowFixMe (tsx)
 import Header from '../Header/Header';
+import HeaderPrototype from '../Header/HeaderPrototype';
```
Comment (jamesgorrie) [inline common/views/components/PageLayoutDeprecated/PageLayoutDeprecated.js:11]: You need a `FlowFixMe`
```suggestion
// $FlowFixMe
import HeaderPrototype from '../Header/HeaderPrototype';
```

**Review by jamesgorrie — APPROVED** [review]
Assuming we'll see the diff when we merge `HeaderPrototype` into `Header`?

You need `$FlowFixMe` the import.

**Comment by gestchild** [comment]
I can't access the links inside the dropdown via a keyboard, looks like the tabindex isn't being reset

**Comment by davidpmccormick** [comment]
> I can't access the links inside the dropdown via a keyboard, looks like the tabindex isn't being reset

Yeah I spotted this, too – I've got a note on my todo list to look into it.

**Pull request merged**